WhaleSpotting/react-app/src/api/apiClient.ts

```diff
@@ -0,0 +1,30 @@
+import authService from "../components/api-authorization/AuthorizeService";
+
+export async  function getHeaders(): Promise<any> {
+    const token = await authService.getAccessToken();
+    return !token ? {} : { "Authorization": `Bearer ${token}` };
+}
```

Comment:
```suggestion
export async function getHeaders(): Promise<any> {
    const token = await authService.getAccessToken();
    return !token ? {} : { "Authorization": `Bearer ${token}` };
}
```
A little too many spaces, and I think we should put this at the bottom of the file with no export :)